exercises/bob/test/bob_test.exs

```diff
@@ -60,16 +85,56 @@ defmodule BobTest do
     assert Bob.hey("  ") == "Fine. Be that way!"
   end
 
+  @tag :pending
+  test "alternate silence" do
+    assert Bob.hey("\t\t\t\t\t\t\t\t\t\t") == "Fine. Be that way!"
+  end
+
   @tag :pending
   test "only numbers" do
     assert Bob.hey("1, 2, 3") == "Whatever."
   end
 
+  @tag :pending
+  test "multiple line question" do
+    assert Bob.hey("\nDoes this cryogenic chamber make me look fat?\nNo.") == "Whatever."
+  end
+
   @tag :pending
   test "question with numbers" do
     assert Bob.hey("4?") == "Sure."
   end
 
+  @tag :pending
+  test "non-letters with question" do
+    assert Bob.hey(":) ?") == "Sure."
+  end
+
+  @tag :pending
+  test "prattling on" do
+    assert Bob.hey("Wait! Hang on. Are you going to be OK?") == "Sure."
+  end
+
+  @tag :pending
+  test "starting with whitespace" do
+    assert Bob.hey("         hmmmmmmm...") == "Whatever."
+  end
+
+  @tag :pending
+  test "ending with whitespace" do
+    assert Bob.hey("Okay if like my  spacebar  quite a bit?   ") == "Whatever."
```

Comment:
I was doing some quick spot-checking and it looks like this [should be returning](https://github.com/exercism/problem-specifications/blob/master/exercises/bob/canonical-data.json#L185-L187) `"Sure."` instead of `"Whatever."`.

Reply:
It would be nice to have some automated generators that consume `canonical-data.json`s so that this becomes a non-problem. That's probably beyond the scope of this PR.

Reply:
Hmm... I can fix that, it's likely a copy/paste error. It is surprising that it passed the tests.

It would be great to have a generator, I also think it's beyond the scope of this PR. :)

Reply:
This is now fixed. Thanks for catching it.